### PR TITLE
chore(linter): change to prospector

### PR DESCRIPTION
**Pull request by portellaa**

### CODE REVIEW

File: .prospector.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  options:
+    max-locals: 15
+    max-returns: 6
+    max-branches: 12
+    max-statements: 50
+    max-parents: 7
+    max-attributes: 7
+    min-public-methods: 2
+    max-public-methods: 20
+    max-module-lines: 1000
```
Comment (jfsantos-ds) [inline .prospector.yaml:22]: We probably could lower this one to 500

File: .prospector.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  - too-few-public-methods
+
+  options:
+    max-locals: 15
+    max-returns: 6
+    max-branches: 12
+    max-statements: 50
+    max-parents: 7
+    max-attributes: 7
+    min-public-methods: 2
```
Comment (jfsantos-ds) [inline .prospector.yaml:20]: This will count inherited methods?

Comment (portellaa) [reply .prospector.yaml:20]: not sure about that, but i assume it will.

**Review by jfsantos-ds — COMMENTED** [review]
Fine by me, just in doubt about 2 thresholds, min public methods (unless it counts parent class methods) and max module size in lines of code (1000 seems like way to much to be of use)

**Comment by portellaa** [comment]
either way, this means that all the warning and messages that prospector reports, has to be fixed, otherwise it will not pass

**Pull request merged**